src/commons/components/info-banner/InfoBanner.vue

```diff
@@ -77,23 +82,24 @@ export default defineComponent({
   },
   props: {
     eventBanner: {
-      type: Object as () => {
-        bannerImg: string;
-        bannerImgHash: string;
-        profileImg: string;
-        profileImgHash: string;
-        title: string;
-        tags: string[];
-        date: string;
-        time: string;
-        location: string;
-      },
-      required: true
+      type: Object as () => GetEventByIdQuery["event"]
     },
     hasButton: {
       type: Boolean,
       defualt: false

```

Comment:
Can we change this naming? Maybe to something like 'canRegister' because 'hasButton' does not actually reflect what button does.